### DISCUSSION TITLE
Question about gameplay

### DISCUSSION THREAD

**Discussion by lafse** [OpenApoc/OpenApoc#1179] [Q&A]
First of all, I wanted to say thank you for the old school revival. The game is wonderful and I will be happy to pass it, thanks to you!

My question: I remember that if you complete all tactical quests to destroy aliens on the ground- organizations do not come under the control of aliens. Passing openapoc, I completed each of the quests, but organizations are coming under control and i can't do anything with it. Am I doing something wrong? Or is it a bug?

**Comment by makus82** [comment]
Hey @lafse , thx! main discuss going in discord for now
Also this seems bug or dev setting to check alien infiltration and moves...

https://github.com/OpenApoc/OpenApoc#contact-us
